rocAL/rocAL_pybind/examples/rocAL_api_python_unittest.py

```diff
@@ -27,20 +27,45 @@
 import amd.rocal.types as types
 from parse_config import parse_args
 import os
+import sys
+import cv2
 import cupy as cp
 
+INTERPOLATION_TYPES = {
+    0: types.NEAREST_NEIGHBOR_INTERPOLATION,
+    1: types.LINEAR_INTERPOLATION,
+    2: types.CUBIC_INTERPOLATION,
+    3: types.LANCZOS_INTERPOLATION,
+    4: types.GAUSSIAN_INTERPOLATION,
+    5: types.TRIANGULAR_INTERPOLATION
+}
+
+SCALING_MODES = {
+    0: types.SCALING_MODE_DEFAULT,
+    1: types.SCALING_MODE_STRETCH,
+    2: types.SCALING_MODE_NOT_SMALLER,
+    3: types.SCALING_MODE_NOT_LARGER
+}
+
+
 def draw_patches(img, idx, device):
-    #image is expected as a tensor, bboxes as numpy
-    import cv2
+    # image is expected as a tensor, bboxes as numpy
     args = parse_args()
     if device == "gpu":
         img = cp.asnumpy(img)
-    if not args.NHWC:
```

Comment:
Why is this case that handles NHWC removed ?

We need to be handling this too. Add it back

Reply:
We only handle NHWC cases in the python unit tests. In case of grayscale images, the tensor format is NCHW but for that we are checking if its grayscale and then transposing it

Reply:
Unit test should test both NCHW & NHWC right?

Reply:
Shobana asked me to exactly port the C++ unit tests to python and since C++ unit tests doesn't have these test cases, we are not handling it here also